install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -220,6 +224,40 @@ If Kuryr is enabled, {product-title} services are implemented through OpenStack
 Octavia does not support UDP load balancing. Services that expose UDP ports are not supported.
 ====
 
+[NOTE]
+====
+The `kuryr_openstack_global_namespace` defines the list of namespaces that are
+global and can be accessed from any other namespace, as well as access any
+namespace. By default only the `default` and the `openshift-monitoring`
+namespaces are included in the list.
+====
+
+[IMPORTANT]
+====
+For upgrades from previous z-streams, note the access to the other namespaces
```

Comment:
Same:
> If you are upgrading from a previous z-release of {product-title} {product-version}, note that access to other 
>namespaces from global namespaces is controlled by the security group `*-allow_from_default`. 
>
> Although the `remote_group_id` rule can control access to other namespaces from global namespaces, using it can 
> cause scaling and connectivity problems. To avoid these problems, switch from using `remote_group_id` to >`*_allow_from_default`.

I'll probably also turn this into its own subsection. It's a little long to be contained in an admonition.